src/Provider/AbstractProvider.php

```diff
@@ -74,43 +74,21 @@ public function getHttpClient()
         return $client;
     }
 
-    /**
-     * Get the URL that this provider uses to begin authorization.
-     *
-     * @return string
-     */
+    // ProviderInterface
```

Comment:
That's a common misconception, and not what `inherticdoc` is meant for. As per [the documentation](http://www.phpdoc.org/docs/latest/guides/inheritance.html):

> Currently some applications have DocBlocks containing just the {@inheritDoc} inline tag to indicate that their complete contents should be inherited. This usage breaks with the PHPDoc Standard as summaries cannot contain inline tags and inheritance is automatic; you do not need to define a special tag for it.